apps/projects/app/components/Content/IssueDetail/BountyCard.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  Box,
+  Button,
+  GU,
+  IconAddUser,
+  IconClock,
+  IconFile,
+  IconGraph2,
+  IdentityBadge,
+  Link,
+  Text,
+  useTheme,
+} from '@aragon/ui'
+import { issueShape } from '../../../utils/shapes.js'
+import { usePanelManagement } from '../../Panel'
+import { useAragonApi } from '../../../api-react'
+import { formatDistance } from 'date-fns'
+import { IconDoubleCheck, IconUserCheck } from '../../../assets'
+
+const ActionButton = ({ panel, caption, issue }) => (
+  <EventButton
+    mode="normal"
+    wide
+    onClick={() => panel(issue)}
+  >
+    <Text size="large">
+      {caption}
+    </Text>
+  </EventButton>
+)
+ActionButton.propTypes = {
+  panel: PropTypes.func.isRequired,
+  caption: PropTypes.string.isRequired,
+  issue: PropTypes.oneOfType([
+    issueShape,
+    PropTypes.arrayOf(issueShape),
```

Comment:
An array of issues was added as special case for funding bounties. Funding panel requires an array (because user can select and fund multiple issues in one go). For a moment we showed "fund issue" button in the widget - it is no longer the case though, now the widget is (again) visible only for existing bounties. Therefore I've removed the `oneOfType` entirely.